.github/workflows/e2e.yaml

```diff
@@ -69,56 +86,115 @@ jobs:
           name: terraform-provider-clickhousedbops
           path: ${{ env.TMPDIR }}
 
-  e2e:
+  database:
```

Comment:
Might be worth adding a prefix to these test names? e.g. e2e_database, e2e_role or even
run_e2e_database, run_e2e_role

In general I expected that all job names are verbs. 'database' is kind of easy to confuse with some other job that could prepare a DB for something unrelated to tests

Reply:
this is a good suggestion, but with the goal of minimizing duplicate code, I'm using the job name as the test folder to be tested. So all the single jobs are identical, apart from the ID.